python/lib/cloudutils/serviceConfig.py

```diff
@@ -452,6 +454,7 @@ def config(self):
             cfgline = "root:/usr/sbin/libvirtd  cpu virt/\n"
             cfo.add_lines(cfgline)
 
+            # TODO: This doesn't work on EL10/Fedora 40
             self.syscfg.svo.stopService("cgred", True)
```

Comment:
This isn't important or urgent at all, but we expect EL10 to be public by mid-2025.
@weizhouapache @shwstppr @DaanHoogland @Pearl1594 do you know how/why cgroup setup code is run when `cloudstack-setup-agent` is called when adding KVM host to ACS? The `cgred` service is missing on Fedora 40 / EL 10.

Reply:
@rohityadavcloud , i'm not sure if it is equivilent (and if so it might not be a drop in replacement) but cgconfig.service is the fedora way of configuring cgroups.

I'll be reading up on it and updating here.